### PR TITLE
feat: Estimate when signing mint

### LINKED ISSUE #9370: Estimate when signing mint
image
- there is no estimate, can we please add it after:

- #9165

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #9370

## Needs QA check

- @kodadot/qa-guild please review

## Needs Design check

- @exezbcz please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

## Paid Mint 

https://github.com/kodadot/nft-gallery/assets/44554284/f6948f68-fc79-4ee1-84fd-7f6d09e6189e

## Free Mint

https://github.com/kodadot/nft-gallery/assets/44554284/7482c02e-673a-4151-99ad-deadca78d62a

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

File: components/shared/SigningModal/SigningModalBody.vue [hunk 1]
```diff
@@ -39,27 +39,51 @@
...
 defineEmits(['tryAgain'])
-withDefaults(
+const props = withDefaults(
   defineProps<{
     title: string
     subtitle: string
     showSubtitleDots?: boolean
     failed?: boolean
+    status?: TransactionStatus
+    customFormattedEstimation?: string
```
Comment (preschian) [inline components/shared/SigningModal/SigningModalBody.vue:68]: eslint, add default value

**Review by preschian — APPROVED** [review]

File: components/shared/SigningModal/SigningModalBody.vue (hunk 1 above)
Comment (prury) [reply components/shared/SigningModal/SigningModalBody.vue:68]: @hassnian

Comment (hassnian) [reply components/shared/SigningModal/SigningModalBody.vue:68]: fixed f7c0f2b

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 10.65 USD/DOT ~ 4.695 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0xda4f3fe58593f573e1586f0b4b580363ff8680b4f393e969d29fbedc451ac40d](https://polkadot.subscan.io/extrinsic/0xda4f3fe58593f573e1586f0b4b580363ff8680b4f393e969d29fbedc451ac40d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**